packages/web-components/fast-foundation/src/tree-item/tree-item.template.ts

```diff
@@ -35,31 +39,34 @@ export function treeItemTemplate(
                 filter: elements(),
             })}
         >
-            <div class="positioning-region" part="positioning-region">
-                <div class="content-region" part="content-region">
-                    ${when(
-                        x => x.childItems && x.childItemLength(),
-                        html<FASTTreeItem>`
-                            <div
-                                aria-hidden="true"
-                                class="expand-collapse-button"
-                                part="expand-collapse-button"
-                                @click="${(x, c) =>
-                                    x.handleExpandCollapseButtonClick(
-                                        c.event as MouseEvent
-                                    )}"
-                                ${ref("expandCollapseButton")}
+            <div class="treeitem" part="treeitem">
+                ${when(
+                    x => x.childItems && x.childItemLength(),
+                    html<FASTTreeItem>`
+                        <div
+                            aria-hidden="true"
+                            class="expand-collapse-button"
+                            part="expand-collapse-button"
+                            @click="${(x, c) =>
+                                x.handleExpandCollapseButtonClick(c.event as MouseEvent)}"
+                            ${ref("expandCollapseButton")}
+                        >
+                            <span
+                                class="expand-collapse-icon"
+                                part="expand-collapse-icon"
                             >
-                                <slot name="expand-collapse-glyph">
-                                    ${options.expandCollapseGlyph || ""}
+                                <slot name="expand-collapse-icon">
+                                    ${options.expandCollapseIcon || expandCollapseIcon}
                                 </slot>
-                            </div>
-                        `
-                    )}
-                    ${startSlotTemplate(options)}
+                            </span>
+                        </div>
+                    `
+                )}
+                ${startSlotTemplate(options)}
+                <span class="content" part="content">
```

Comment:
Assuming this is necessary for truncation purposes? If truncation is already doable, is this just for the sake of "consistency"? If that's the case, I'm not sure i'm a fan of bloating templates because everything needs to be the same, etc. Templates should represent the most common scenarios in as efficient a way as possible and if we're adding DOM just to be "similar" that's going to have implications to performance where we could have smaller templates, require less dom, less css selectors, etc...